model/registry/messaging.yaml

```diff
@@ -235,6 +235,21 @@ groups:
         brief: >
          The ordering key for a given message. If the attribute is not present, the message does not have an ordering key.
         examples: 'ordering_key'
+      - id: gcp_pubsub.message.ack_id
+        type: string
+        brief: >
+         The ack id for a given message.
+        examples: 'ack_id'
+      - id: gcp_pubsub.ack_deadline_seconds
```

Comment:
OTel doesn't recommend adding units to attribute names (I can't find the discussion, but went through it when discussing message|envelope|body size. 

I suggest removing seconds from the name and instead mentioning it in the brief.
Also left a comment below about namespace.

Reply:
https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto#L1403 

The seconds is included in the parameter name.

@hongalex What do you think?

Reply:
Can't this information be added as a note/description on the attribute and leave the actual name without `_seconds`?

Reply:
I think it makes sense, but it is part of our API to have seconds in the name, so I'm not sure what we should do here. I asked Alex to take a look when he gets the chance.

Reply:
Discussed with Alex. Removing seconds is fine.

Reply:
Removed seconds in the PR